staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -125,6 +127,10 @@ func Compile(s *schema.Structural, declType *apiservercel.DeclType, perCallLimit
 	if len(s.XValidations) == 0 {
 		return nil, nil
 	}
+	if declType == nil {
+		klog.Errorf("Failed to convert to declType for CEL validation rules")
+		return nil, nil
```

Comment:
Shouldn't this return nil, err instead of logging?

Reply:
ops. Updated :)